documentation/tutorials/getting-started-with-graphql.md

```diff
@@ -149,3 +149,16 @@ end
 If you started with `mix new ...` instead of `mix phx.new ...` and you want to
 still use phoenix, the fastest path that way is typically to just create a new
 phoenix application and copy your resources/config over.
+
+## What's next?
```

Comment:
This is great, I didn't realize we didn't even have a section for what to look at next. Thank you for adding this and adding more links! Great work.